apps/src/lab2/utils/notifyLevelChange.ts

```diff
@@ -0,0 +1,13 @@
+import Lab2Registry from '../Lab2Registry';
+
+import {LifecycleEvent} from './LifecycleNotifier';
+
+// Function for external components to notify the Lab2 lifecycle notifier that the level is changing.
+export default function notifyLevelChange(
```

Comment:
Not sure if this file is overkill; I mostly felt weird about importing `Lab2Registry` and `LifecycleNotifier` in files outside of lab2 (like `progressRedux` and `browserNavigation`)